agent/tcs/handler/handler_test.go

```diff
@@ -141,7 +141,7 @@ func TestStartSession(t *testing.T) {
 func TestSessionConnectionClosedByRemote(t *testing.T) {
 	// Start test server.
 	closeWS := make(chan []byte)
-	server, serverChan, _, serverErr, err := mockwsutils.StartMockServer(t, closeWS)
+	server, serverChan, _, serverErr, err := utils.StartMockServer(t, closeWS)
```

Comment:
Yay for fixing the package name. However do you think it would be clearer to import that `util` package under a more accurate name like `wsmock` so the mock server is started with `wsmock.StartMockServer` which tells us what mock server is starting?